### PR TITLE
[front] - fix(AddToSpaceDialog): make content scrollable

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at making the content of the `AddToSpaceDialog` scrollable.

## Risk

Low

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/spaces/AddToSpaceDialog.tsx [hunk 1]
```diff
@@ -153,14 +155,28 @@ export const AddToSpaceDialog = ({
...
+              hideScrollBar
+            >
+              {[
+                ...availableSpaces,
+                ...availableSpaces,
+                ...availableSpaces,
+                ...availableSpaces,
+                ...availableSpaces,
+                ...availableSpaces,
+                ...availableSpaces,
```
Comment (aubin-tchoi) [inline front/components/spaces/AddToSpaceDialog.tsx:170]: not sure I understand that

**Review by aubin-tchoi — APPROVED** [review]
LGTM

File: front/components/spaces/AddToSpaceDialog.tsx [hunk 2]
```diff
@@ -153,14 +155,20 @@ export const AddToSpaceDialog = ({
...
-            {availableSpaces.map((currentSpace) => (
-              <DropdownMenuItem
-                key={currentSpace.sId}
-                label={currentSpace.name}
-                onClick={() => setSpace(currentSpace)}
-              />
-            ))}
+          <DropdownMenuContent className="min-w-[200px] p-2">
+            <ScrollArea
+              className="flex max-h-[300px] flex-col overflow-y-auto"
```
Comment (flvndvd) [inline front/components/spaces/AddToSpaceDialog.tsx:160]: Let's avoid arbitrary values please 🙏. You can use either `h-72` or `h-80`.

Comment (JulesBelveze) [reply front/components/spaces/AddToSpaceDialog.tsx]: Couldn't figure out why `min-w-*` classes weren't available. But modified the tailwindconfig and it should work now

**Pull request merged**